### DISCUSSION TITLE
Bazel and MapLibre Native iOS development experiences

### DISCUSSION THREAD

**Discussion by louwers** [maplibre/maplibre-native#1145] [General]
Creating a thread to gather feedback on using Bazel to set up the Xcode project for MapLibre Native iOS development.

Documentation how to use Bazel to set up Xcode part of this PR: https://github.com/maplibre/maplibre-native/pull/1146/files#diff-ee03e3ca243c6329eb9a6520498434b1d36c5c689d66996396f73211dee44d9b

**Comment by birkskyum** [comment]
It would appear that this Supported Platforms setting is set to iphonesimulator when the xcode proj is created. I had to change it to iOS in order to use a physical device (iphonesimulator isn't an option in the dropdown, so maybe that value is obsolete..). Would be good to have the bazel script set it to iOS by default.

<img width="1261" alt="Screenshot 2023-05-24 at 01 02 15" src="https://github.com/maplibre/maplibre-native/assets/74932975/4f6ed737-59ac-4235-b498-5cebe97f7a56">
